diff --git a/helpers/runner_wrapper/wrapper_unix.go b/helpers/runner_wrapper/wrapper_unix.go
index 659794283..4a5945c4a 100644
--- a/helpers/runner_wrapper/wrapper_unix.go
+++ b/helpers/runner_wrapper/wrapper_unix.go
@@ -1,4 +1,4 @@
-//go:build aix || android || darwin || dragonfly || freebsd || hurd || illumos || linux || netbsd || openbsd || solaris
+//go:build aix || android || darwin || dragonfly || freebsd || hurd || illumos || linux || netbsd || openbsd || solaris || zos
 
 package runner_wrapper
 
