diff --git a/shells/bash.go b/shells/bash.go
index 33c18a28c..8f6b32abe 100644
--- a/shells/bash.go
+++ b/shells/bash.go
@@ -360,13 +360,13 @@ func (b *BashWriter) EmptyLine() {
 
 func (b *BashWriter) SectionStart(id, command string, options []string) {
 	b.Line("printf '%b\\n' " +
-		"section_start:$(date +%s):section_" + id + stringifySectionOptions(options) +
+		"section_start:${EPOCHSECONDS}:section_" + id + stringifySectionOptions(options) +
 		"\r" + helpers.ANSI_CLEAR + b.escape(helpers.ANSI_BOLD_GREEN+command+helpers.ANSI_RESET))
 }
 
 func (b *BashWriter) SectionEnd(id string) {
 	b.Line("printf '%b\\n' " +
-		"section_end:$(date +%s):section_" + id +
+		"section_end:${EPOCHSECONDS}:section_" + id +
 		"\r" + helpers.ANSI_CLEAR)
 }
 
